apps/next/app/actions/create-post/page.tsx

```diff
@@ -68,7 +68,7 @@ export default function CreatePostPage({
         <CircleHelp className="h-4 w-4" />
         <AlertTitle className="font-bold">Post anonymously to Farcaster</AlertTitle>
         <AlertDescription>
-          Must have <b>5,000 $RUMOUR</b> in your wallet to post. Posts are made anonymous
+          Must have <b>10,000,000 $RUMOUR</b> in your wallet to post. Posts are made anonymous
```

Comment:
Where does this show? I can't find it in the current UI

Reply:
Not sure, but changed it anyway just in case.